applications/app/views/fragments/crosswords/crosswordContent.scala.html

```diff
@@ -5,46 +5,49 @@
 @import conf.switches.Switches.CommercialSwitch
 @import views.support.Commercial.shouldShowAds
 
-<div class="l-side-margins">
-    <article id="crossword" class="content content--article tonal tonal--tone-news" role="main">
-        @crosswordMetaHeader(crosswordPage)
+@defining(CommercialSwitch.isSwitchedOn && shouldShowAds(crosswordPage)) { adsEnabled =>
+    <div class="l-side-margins">
+        <article id="crossword" class="content content--article tonal tonal--tone-news" role="main">
+            @crosswordMetaHeader(crosswordPage)
 
-    <div class="content__main tonal__main tonal__main--tone-news">
-        <div class="gs-container">
-            <div class="js-content-main-column">
+            <div class="content__main tonal__main tonal__main--tone-news @if(adsEnabled) {crossword--with-ad} else {}">
+                <div class="gs-container">
+                    <div class="js-content-main-column">
+                        <div class="@if(adsEnabled) {crossword__spacer--ad}">
```

Comment:
Random Play template question:

Line 13 uses: `@if(adsEnabled) {crossword--with-ad} else {}` ...why does this need the `else {}` while it seems okay to omit it here ? `@if(adsEnabled) {crossword__spacer--ad}`

Reply:
It doesn't, you're right! Inconsistent tidying up. Great spot, thanks.

Reply:
In fact, crossword-with-ads wasn't supposed to be there at all (intermediate testing) so doubly well spotted, thanks again.